web/src/app/schedules/ScheduleOverrideCreateDialog.tsx

```diff
@@ -49,64 +63,56 @@ const mutation = gql`
     }
   }
 `
-export default function ScheduleOverrideCreateDialog(props) {
+export default function ScheduleOverrideCreateDialog(
+  props: ScheduleOverrideCreateDialogProps,
+): JSX.Element {
+  const {
+    scheduleID,
+    variant,
+    onClose,
+    removeUserReadOnly,
+    defaultValue = {},
+  } = props
```

Comment:
we could destructure this inside of the function params and get rid of using `props` as a variable all together

```
export default function ScheduleOverrideCreateDialog({
    scheduleID,
    variant,
    onClose,
    removeUserReadOnly,
    defaultValue = {},
  }: ScheduleOverrideCreateDialogProps,
): JSX.Element {
  
```